autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/feature_preprocessing/RandomKitchenSinks.py

```diff
@@ -35,35 +36,38 @@ def fit(self, X: Dict[str, Any], y: Any = None) -> BaseEstimator:
     @staticmethod
     def get_hyperparameter_search_space(
         dataset_properties: Optional[Dict[str, str]] = None,
-        n_components: Tuple[Tuple, float, bool] = ((0.5, 0.9), 0.5, True),
-        gamma: Tuple[Tuple, float, bool] = ((3.0517578125e-05, 8), 1.0, True),
-        degree: Tuple[Tuple, int] = ((2, 5), 3),
-        coef0: Tuple[Tuple, float] = ((-1, 1), 0)
+        n_components: HyperparameterSearchSpace = HyperparameterSearchSpace(hyperparameter='n_components',
+                                                                            value_range=(0.5, 0.9),
+                                                                            default_value=0.5,
+                                                                            ),
+        gamma: HyperparameterSearchSpace = HyperparameterSearchSpace(hyperparameter='gamma',
+                                                                     value_range=(3.0517578125e-05, 8),
+                                                                     default_value=0.01,
```

Comment:
Shouldn't the default be 1.0?

Reply:
yes, thanks for catching this. I must have messed up while changing